src/snyk/snykCode/views/suggestion/codeSuggestionWebviewProvider.ts

```diff
@@ -213,6 +280,78 @@ export class CodeSuggestionWebviewProvider
     }
   }
 
+  private highlightAddedCode(fileUri: string, patch: string) {
+    const highlightDecoration = vscode.window.createTextEditorDecorationType({
+      backgroundColor: 'rgba(0,255,0,0.3)',
```

Comment:
this needs to be theme compatible

Reply:
look into if something here makes sense to use: https://code.visualstudio.com/api/references/theme-color